packages/react-icon-provider/package.json

```
@@ -0,0 +1,50 @@
+{
+  "name": "@fluentui/react-icon-provider",
+  "version": "0.1.0",
+  "description": "Package for applying icon overrides to new svg icons",
+  "private": true,
+  "main": "lib-commonjs/index.js",
+  "module": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "sideEffects": [
+    "lib/version.js"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui"
+  },
+  "license": "MIT",
+  "scripts": {
+    "build": "just-scripts build",
+    "bundle": "just-scripts bundle",
+    "clean": "just-scripts clean",
+    "code-style": "just-scripts code-style",
+    "just": "just-scripts",
+    "lint": "just-scripts lint",
+    "start": "just-scripts dev:storybook",
+    "test": "just-scripts test",
+    "update-api": "just-scripts update-api"
```

You can remove `update-api` since it happens automatically now (we should also remove this from the template and all the other packages later)
```suggestion
    "test": "just-scripts test"
```